nodes/src/nodes/llm_openai_api/IGlobal.py

```
@@ -0,0 +1,96 @@
+                        if status:
+                            parts.append(f'Error {status}:')
+                        if etype:
+                            parts.append(etype)
+                        if emsg:
+                            if etype:
+                                parts.append('-')
+                            parts.append(emsg)
+                        if parts:
+                            message = ' '.join(parts)
+                except Exception:
+                    pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)